process_manager/main.py

```diff
@@ -0,0 +1,66 @@
+
+import argparse
+import logging
+import asyncio
+import signal
+from datetime import datetime, timedelta
+from random import random, randint
+from task import DEFAULT_PERIOD, DEFAULT_POOL_SIZE, DEFAULT_TIMEOUT, SchedulerTask
```

Comment:
If we're not using these in `task` but using them here, why not just define them here?